src/main/java/service/FacultyService.java

```diff
@@ -27,7 +27,7 @@ public void delete (long id){
         facultyRepository.deleteById(id);
     }
     public Collection<Faculty> findByColorOrName(String part){
-        return facultyRepository.findByNameContains(part);
+        return facultyRepository.findByNameOrColourIgnoreCase(part);
```

Comment:
Ну и соответсвенно передаём два параметра
```suggestion
        return facultyRepository.findByNameOrColourIgnoreCase(part, part);
```